lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1917,12 +1917,12 @@ def cmd_sessions(*args)
             session.response_timeout = response_timeout
           end
           begin
-            if session.type == 'shell'
+            if session.type.include? 'shell'
```

Comment:
Otherwise the code reader needs to guess one of the session types is 'powershell' :)